content/docs/fairing/_index.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Fairing"
+title = "Kubeflow Fairing"
```

Comment:
Please revert this change to keep "Fairing" as the page title.

In the Kubeflow docs, we leave Kubeflow out of the title for components/features such as Pipelines and Fairing to keep the page navigation concise and easy to use.

Reply:
Rollbacked, thanks @joeliedtke